forms/NullableField.php

```diff
@@ -77,53 +100,80 @@ public function getIsNullId() {
 	}
 
 	/**
-	 * (non-PHPdoc)
-	 * @see framework/forms/FormField#Field()
+	 * @param array $properties
+	 *
+	 * @return string
 	 */
 	public function Field($properties = array()) {
-		if ( $this->isReadonly()) {
+		if($this->isReadonly()) {
 			$nullableCheckbox = new CheckboxField_Readonly($this->getIsNullId());
 		} else {
 			$nullableCheckbox = new CheckboxField($this->getIsNullId());
 		}
+
 		$nullableCheckbox->setValue(is_null($this->dataValue()));
 
-		return $this->valueField->Field() . ' ' . $nullableCheckbox->Field()
-			. '&nbsp;<span>' . $this->getIsNullLabel().'</span>';
+		return sprintf(
+			'%s %s <span>%s</span>',
+			$this->valueField->Field(),
+			$nullableCheckbox->Field(),
+			$this->getIsNullLabel()
+		);
 	}
 
 	/**
 	 * Value is sometimes an array, and sometimes a single value, so we need to handle both cases
+	 *
+	 * @param mixed $value
+	 * @param null|array $data
+	 *
+	 * @return static
```

Comment:
Should we use `$this` return type? static implies the same (late bound) type, but not necessarily the same instance, and is probably more useful for static methods.

In this case I think `$this` adds more value. :D

See http://www.phpdoc.org/docs/latest/guides/types.html for reference.